test-cypress/stubs/ttyg/ttyg-stubs.js

```diff
@@ -178,4 +178,78 @@ export class TTYGStubs extends Stubs {
             });
         }).as('get-answer');
     }
+
+    static stubExplainResponse() {
+        cy.intercept({
+            method: 'POST',
+            url: 'rest/chat/conversations/explain'
+        }, (req) => {
+            const requestBody = req.body;
+
+            const explainResponse = {
+                conversationId: requestBody.conversationId,
+                answerId: requestBody.answerId,
+                queryMethods: [
```

Comment:
You are often using such smarter stubs but I don't like them. They are harder to be maintained and updated.

Reply:
why don't you just use fixtures?

Reply:
but they are closer to how the real system works.

Reply:
Done